robottelo.properties.sample

```diff
@@ -271,3 +271,23 @@ ssh_key=
 # for RHEL7 - the ports needs to have a correct selinux context set:
 # 'semanage port -a -t websm_port_t -p tcp <port-range>'
 port_range=9091, 14999
+
+# section for real capsule
+# [capsule]
+# to setup a real capsule we need the capsule virtual machine hostname
+# resolvable.
+# we should register a hostname and get a hash value as described in
+# https://mojo.redhat.com/docs/DOC-13143, that we should use on the stage of
+# ddns setup
+# domain is the base ddns domain, the virtual machine hostname will be a sub
+# domain of this domain.
+# domain=
+# the immage_name is the key used in the registration above, and used to name
+# the image of the virtual machine
+# the real finale hostname should be image_name.domain
+# image_name=
```

Comment:
I feel we need RHEL7 image of capsule for a satellite6 running on RHEL7 here and RHEL6 image of capsule for a satellite6 running on RHEL6. Even RHEL6 for RHEL7 and viceversa also could work but I think RHEL7 capsule for RHEL7 satellite and RHEL6 capsule for RHEL6 satellite is what we require currently.